modules/core/src/main/scala/com.snowplowanalytics.iglu/client/resolver/registries/RegistryLookup.scala

```diff
@@ -156,15 +90,15 @@ object RegistryLookup {
   private[registries] def toPath(prefix: String, key: SchemaKey): String =
     s"${prefix.stripSuffix("/")}/schemas/${key.toPath}"
 
-  private def toSubpath(
+  private[registries] def toSubpath(
```

Comment:
Could we move `toSubpath` methods to JavaNet... object or are these used by http4s as well?

Reply:
They are used by http4s as well.

Reply:
Hmmm so what about [this](https://github.com/snowplow/iglu-scala-client/blob/4fe1cb96e2c1a1c52a1a7f44f62237b5601cc5be/modules/http4s/src/main/scala/com.snowplowanalytics.iglu/client/resolver/registries/Http4sRegistryLookup.scala#L93) one? Looks like http4s has its own `toSubpath` method...?

Reply:
You were right.  I've changed it now.